include/cloudbreak.bash

```diff
@@ -648,5 +658,19 @@ util-smartsense() {
       -e CB_SMARTSENSE_CONFIGURE=$CB_SMARTSENSE_CONFIGURE \
       -l traefik.enable=false \
       -v $PWD:/var/lib/cloudbreak-deployment \
+      -v $(which cbd):/bin/cbd \
+      -p 9000:9000 \
       $DOCKER_IMAGE_CBD_SMARTSENSE:$DOCKER_TAG_CBD_SMARTSENSE
 }
+
+util-get-usage() {
+    declare desc="Generate Flex related usages."
+
+    cloudbreak-config
+
+    local CRED="$UAA_FLEX_USAGE_CLIENT_ID:$UAA_FLEX_USAGE_CLIENT_SECRET"
+    local CRED_BASE64=$(echo -n ${CRED}|base64)
+    local TOKEN=$(curl -sX POST -H "Authorization: Basic $CRED_BASE64" "${PUBLIC_IP}:${UAA_PORT}/oauth/token?grant_type=client_credentials" | jq '.access_token' -r)
```

Comment:
I think no, because in this case we need to get a token for an Oauth client and not a client on a behalf of a user like util-token function.